### ISSUE TITLE
Finetuning before feature extraction

### ISSUE THREAD

**Issue by Stuffooh** [huggingface/transformers#3271]
Hi,

Currently I am using the pipeline feature extraction to extract features with my own dataset as input. I was wondering if it is possible to finetune different models using my own dataset before using the pipeline for feature extraction and if so what will be the easiest way to do so?

In the past there used to be a lm_finetuning script example but this one is no longer available. I cannot find any examples or guides how to finetune different models on a personal dataset.

tl;dr is it possible to finetune different models on a personal dataset in just a few lines of code and if so how?

Thanks in advance,

A clueless person trying to learn more about the world of NLP.

**Comment by julien-c** [comment]
The script has been renamed [`run_language_modeling.py`](https://github.com/huggingface/transformers/blob/master/examples/run_language_modeling.py) to better reflect the fact that it can also be used to train a new model from scratch.

Let us know if it works.

**Comment by Stuffooh** [comment]
Hi @julien-c  thanks for the quick reply!

That makes sense, I must have missed the fact it got renamed. I have one more question about the "new" script:

How difficult is it make to script compatible for finetuning a  new model like BART? Is it as simple as adding the model to the list inside the script or will it need a lot of workarounds to get it working?
